src/openms/include/OpenMS/CHEMISTRY/ModificationsDB.h

```diff
@@ -80,6 +80,16 @@ namespace OpenMS
       return db_;
     }
 
+    inline static ModificationsDB * getCustomInstance(OpenMS::String unimod_file, OpenMS::String psimod_file)
```

Comment:
> 

For TOPP tools we could also add the path to the different xml files as advanced parameters in TOPPBase. If we then just have to make sure that the first thing we do is instanciate the dbs one after the other in the correct order.

also sounds like an option to me

>  If only modifications need to be added I actually would prefer the second option (while the first would give the guarantee that all code uses the same db).

unfortunately not, as far as I understand, George is having trouble with Unimod being *overannotated* and he wants to supply a slim version of unimod to the program so that it produces something sane. E.g. unimod would have many more residues as being phosphorylated than the canonical STY and this screws with his codes logic. While technically true there are many more resides that can be phosphorylated, we mostly care about the canonical ones.